### PR TITLE
Portfolio: mobile menu

### PR DESCRIPTION
**Pull request by Sonickmumba**
### Mobile menu implementation: My Portfolio

In this pull request, I

- implemented the mobile menu using Javascript.
- Set up the JavaScript (eslint) linter in project directory.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -46,31 +46,31 @@ body {
...
   position: fixed;
   top: 0;
   left: 0;
-  width: 100%;
+  width: 375px;
   height: 100%;
   z-index: 1;
   background-color: rgba(96, 112, 255, 0.9);
+  margin-left: 100px;
 }
```
Comment (HAMMAS-SALEEM) [inline style.css]: ### Your Design
problem
### Figma Template
Figma

- [ ] You can see your design is not fully expanding on screen as the Figma design. :blush: For fixing it please remove the `margin-left: 100px;` to solve this issue :+1:
- [ ] You can see in the Figma that there is a `backdrop-filter: blur(6px)` on the blue screen. Please add it :+1:

### Follow the Quick Fix :point_down:
quick-fix

File: index.html [hunk 2]
```diff
@@ -26,13 +26,13 @@
...
                     <li class="mobile-link">
-                        <a href="#">About</a>
+                        <a href="#about">About</a>
                     </li>
                     <li class="mobile-link">
-                        <a href="#">Contact</a>
+                        <a href="#contact">Contact</a>
                     </li>
                 </ul>
             </div>
```
Comment (HAMMAS-SALEEM) [inline index.html]: ### Figma with correction
Figma (copy)

- [ ] Please add this black bar in your design to follow Figma the template & to make your design look professional :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @Sonickmumba :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Following HTML, CSS, and JavaScript best practices
:heavy_check_mark: Following Project Requirements
:heavy_check_mark: Descriptive PR title
:heavy_check_mark: Descriptive branch name
:heavy_check_mark: Descriptive PR summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions
_See Below Comments_

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @Sonickmumba 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**